modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -434,6 +462,15 @@ public void testUpdateDiscoveryAttributesOfOrganization() throws IOException {
                 .body("attributes[0].values", containsInAnyOrder("xyz.com", "example.com"));
     }
 
+    @DataProvider(name = "checkDiscoveryAttributes")
```

Comment:
Do we need this location change?

Reply:
When I initially added the `checkDiscoveryAttributes` data provider, I decided to keep all data providers in one place. However, the new `filterQueries` data provider would be more readable if kept closer to the test method. Therefore, I also relocated the `checkDiscoveryAttributes` data provider near the test method for consistency and to improve overall readability.